deploy/gk-deploy

```diff
@@ -29,6 +29,8 @@ ABORT=0
 NODES=""
 SKIP_PREREQ=0
 LOAD=0
+ADMIN_KEY=""
+USER_KEY=""
 
 usage() {
   echo -e "USAGE: ${PROG} [-ghv] [-c CLI] [-t <TEMPLATES>] [-n NAMESPACE] [-w <SECONDS>] [--load]
```

Comment:
Add `[--admin-key <ADMIN_KEY>] [--user-key <USER_KEY>]` in the line below just before the log file option.